### PR TITLE
Changes how heretic sacrifices work (again!)

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

Instead of being telwported to an empty white void and having some minor damage dealt to your character you instead are now telwported to a slightly less empty white void and given the ability to choose the varying amoint of damage dealt to your character before being allowed to leave

This reduces the average time spent in the empty white void and allows some player input however you will still be mauled in some form before being returned to the station

Current pr is in a wip state with the total size of brazil as well as the effects, number of effects required to return and etc being nonfinal

# Wiki Documentation

Wip

# Changelog

:cl:  
Experimental: heretic sacrifices now have some agency as to how they will be mauled after being sacrificed
/:cl:

### CODE REVIEW

File: code/modules/antagonists/eldritch_cult/eldritch_effects.dm [hunk 1]
```diff
@@ -291,4 +301,165 @@
...
-	name = pick(prefix) + " " + pick(postfix)
\ No newline at end of file
+	name = pick(prefix) + " " + pick(postfix)
+
+
+/*
+ *
+ * brazil related statuses and effects
+ * used to create a "fun and intuitive gameplay" for people who get sacrificed by heretics
+ * and by that i mean they get to lose stuff
```
Comment (Hopekz) [inline code/modules/antagonists/eldritch_cult/eldritch_effects.dm:311]:
```suggestion
 * brazil related statuses and effects
 * used to create a "fun and intuitive gameplay" for people who get sacrificed by heretics
 * and by that, I mean they get to lose stuff
```

File: code/modules/antagonists/eldritch_cult/eldritch_effects.dm [hunk 2]
```diff
@@ -291,4 +301,165 @@
...
+	///counts how close to escaping brazil the owner is
+	var/penance_left = 15
+	///sacrifices made to reduce penance_left, each is applied when leaving
+	var/list/penance_sources = list()
+	///list of limbs to do stuff to
+	var/list/unspooked_limbs = list(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)
+
+/obj/screen/alert/status_effect/brazil_penance
+	name = "Otherworldly Tarrif"
+	desc = "The things of this place want something from you, you won't be able to leave until enough has been taken."
```
Comment (Hopekz) [inline code/modules/antagonists/eldritch_cult/eldritch_effects.dm]:
```suggestion
	desc = "The things of this place want something from you, and you won't be able to leave until enough has been taken."
```

**Comment by Hopekz** [comment]
I know this probably doesn't really matter in the grand scheme of the codebase but IMO you should choose one style of comments and stick to it. 

You seem to be jumping back and forth between same line comments and above the line comments which make them harder to read. I recommend same line but it's a personal preference.

**Comment by KittyNoodle** [comment]
make the traumas permanent or people are just gonna put all their points into traumas then rush into medbay to get them cured

**Comment by SomeguyManperson** [comment]
> make the traumas permanent or people are just gonna put all their points into traumas then rush into medbay to get them cured

There's a cap of 3 severe traumas and 1 deep rooted trauma, with deep rooted traumas having a high chance to be replaced with a random permanent trauma if removed surgically. I thought about making them permanent but some traumas are extremely painful to deal with and their current worth is the same as the equally fixable limb damaging effects

**Comment by tattax** [comment]
good

**Comment by JamieD1** [comment]
@wejengin2

**Comment by JamieD1** [comment]
Map Looks good

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**